Android/gradle.properties

```diff
@@ -0,0 +1,63 @@
+# Project-wide Gradle settings.
+# IDE (e.g. Android Studio) users:
+# Gradle settings configured through the IDE *will override*
+# any settings specified in this file.
+# For more details on how to configure your build environment visit
+# http://www.gradle.org/docs/current/userguide/build_environment.html
+# Specifies the JVM arguments used for the daemon process.
+# The setting is particularly useful for tweaking memory settings.
+org.gradle.jvmargs = -Xmx2048m \
+  -Dfile.encoding=UTF-8 \
+  --add-exports=java.base/sun.nio.ch=ALL-UNNAMED \
+  --add-opens=java.base/java.lang=ALL-UNNAMED \
+  --add-opens=java.base/java.lang.reflect=ALL-UNNAMED \
+  --add-opens=java.base/java.io=ALL-UNNAMED \
+  --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED \
+  --add-exports=jdk.unsupported/sun.nio.ch=ALL-UNNAMED
+
+# When configured, Gradle will run in incubating parallel mode.
+# This option should only be used with decoupled projects. More details, visit
+# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
+# org.gradle.parallel=true
+# AndroidX package structure to make it clearer which packages are bundled with the
+# Android operating system, and which are packaged with your app"s APK
+# https://developer.android.com/topic/libraries/support-library/androidx-rn
+android.useAndroidX = true
+
+# Automatically convert third-party libraries to use AndroidX
+android.enableJetifier = true
+
+kotlin.code.style = official
+
+# https://developer.android.com/studio/releases/gradle-plugin
+androidGradlePluginVersion = 7.3.1
+
+# https://github.com/FasterXML/jackson/wiki/Jackson-Releases
+jacksonVersion = 2.13.3
+
+# https://github.com/junit-team/junit4/releases
+junitVersion = 4.13.2
+
+# https://github.com/google/truth/releases
+truthVersion = 1.1.3
+
+kotlinCoroutinesVersion = 1.6.4
+coreVersion = 1.8.0
+
+# https://developer.android.com/studio/releases/gradle-plugin
+androidPluginVersion = 7.3.1
+
+# https://docs.gradle.org/current/release-notes.html
+gradleWrapperVersion = 7.5.1
+
+# Best to use the same Kotlin version that's built into Gradle, otherwise things get messy.
+# https://docs.gradle.org/7.5.1/userguide/compatibility.html
+kotlinVersion = 1.6.21
+
+# Appcompat
+# https://developer.android.com/jetpack/androidx/releases/appcompat
+appcompatVersion = 1.4.2
+
+# AndroidX Lifecycle
+# https://developer.android.com/jetpack/androidx/releases/lifecycle
+lifecycleVersion = 2.5.1
```

Comment:
add this line to the end of this file:

`org.gradle.unsafe.configuration-cache=true`